app/src/main/java/com/algo/phantoms/algo_phantoms/utils/MSPTextViewBold.kt

```diff
@@ -0,0 +1,17 @@
+package com.algo.phantoms.algo_phantoms.utils
```

Comment:
fonts can be directly applied to TextView by styles, or using font attributes, no need of this custom text view